manifests/docker.pp

```diff
@@ -18,6 +18,7 @@
   $shell = $facts['os']['family'] ? {
     'Debian' => '/usr/sbin/nologin',
     'RedHat' => '/sbin/nologin',
+    'Suse'   => '/sbin/nologin',
```

Comment:
maybe combine the two with the same wvalue? `/(RedHat|Suse)/ => '/sbin/nologin',`